src/main/java/io/jenkins/plugins/analysis/core/steps/IssuesRecorder.java

```diff
@@ -435,6 +437,34 @@ public String getReferenceJobName() {
         return referenceJobName;
     }
 
+    /**
+     * Sets the reference build id to get the results for the issue difference computatation.
+     *
+     * @param referenceBuildId
+     *         the build id of the reference job
+     */
+    @DataBoundSetter
+    public void setReferenceBuildId(final String referenceBuildId) {
+        if (NO_REFERENCE_BUILD.equals(referenceBuildId)) {
+            this.referenceBuildId = StringUtils.EMPTY;
+        } else {
```

Comment:
newline before else or remove the else as it is redundant anyway

Reply:
Newline before else added.